okhttp/build.gradle

```diff
@@ -90,6 +90,18 @@ task japicmp(type: me.champeau.gradle.japicmp.JapicmpTask, dependsOn: 'jar') {
       'okhttp3.Cookie$Builder#path(java.lang.String)',
       'okhttp3.Cookie$Builder#secure()',
       'okhttp3.Cookie$Builder#value(java.lang.String)',
+      'okhttp3.Address#certificatePinner()',
+      'okhttp3.Address#connectionSpecs()',
+      'okhttp3.Address#dns()',
+      'okhttp3.Address#hostnameVerifier()',
+      'okhttp3.Address#protocols()',
+      'okhttp3.Address#proxy()',
+      'okhttp3.Address#proxyAuthenticator()',
+      'okhttp3.Address#proxySelector()',
+      'okhttp3.Address#sslSocketFactory()',
+      'okhttp3.Address#url()',
+      'okhttp3.Address#socketFactory()',
+      'okhttp3.TlsVersion#javaName()',
```

Comment:
tiny merge conflict here; this is repeated

Reply:
Woops. Fixed in bacfcb2e26b952bbd5f34c82d137a73646998509